WebGUI/js/MacroMaker.js

```diff
@@ -680,7 +680,7 @@
     {
     	 var fecListEl = document.getElementById("fecList");
     	 var macroLibEl = document.getElementById("macroLib");
-    	 var sequenceEl = document.getElementById("sequence");
+    	 var sequenceEl = document.getElementById("FEsequence");
```

Comment:
What does this fix?

Reply:
I reviewed MacroMaker.html and MacroMaker.js and found there is no other mention of FEsequence html elements. So I can confirm this needs to be reverted.